core/web/schema/schema.graphql

```diff
@@ -10,7 +10,7 @@ schema {
 type Query {
     bridge(id: ID!): BridgePayload!
     bridges(offset: Int, limit: Int): BridgesPayload!
-    chain(id: ID!): ChainPayload!
+    chain(id: ID!, network: Network): ChainPayload!
```

Comment:
can we flip the argument order? Or would it be possible to make `network` part of ID?

Reply:
> Or would it be possible to make network part of ID?

That would be an API breaking change right? Which i wanted to avoid 

Also with the ordering, i believe for graphql the convention is for ID to be the first which unfortunately is chainID in this case `id: ID!`